Level Solutions/31904.0.txt

```diff
@@ -0,0 +1,56 @@
+@0
+
+
+#Checking  the number and then sending it down
+@1
+S: MOV UP, ACC
+JEZ Y
+MOV 1, RIGHT
+MOV ACC, DOWN
+JMP S
+Y: MOV 0, RIGHT
+MOV 0, DOWN
+
+#Add up the number of inputs we have as well as adding the +1 onto the number that is passing through

```

Comment:
Slightly misleading comment -- you're adding 1 to an internal counter, not to the number that is "passing through"